modules/nw-networkpolicy-about.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// networking/configuring-networkpolicy.adoc
+
+[id="nw-networkpolicy-about-{context}"]
+
+= About network policy
+
+In {product-title} {product-version} `NetworkPolicy` is the default mode for the
+`OpenShiftSDN` plug-in for the Kubernetes Container Network Interface (CNI).
```

Comment:
Same question on if this needs to be in backticks. The other places in this PR don't, and I don't think it's necessary.